src/Nri/Ui/TextInput/V5.elm

```diff
@@ -68,7 +68,7 @@ text =
         }
 
 
-{-| An input that allows number entry
+{-| An input that allows integer entry
 -}
 number : InputType (Maybe Int)
```

Comment:
Right now this PR is not a breaking change since it only adds functionality. We could make a breaking change if we feel weird about there being a "number" input separate from a "float" input. @ckoster22 and I talked about this and saw two solutions--either change the name "number" to "integer" or create a custom Number type that encompasses both Int and Float and does some logic as far as Stringify functions go. Does anyone feel strongly about doing this a certain way?

Reply:
seems fine to me ~ maybe in v6 we should make this take a `number` as it's parameter and reconsolidate? Isn't `number` an encapsulating type in elm already?